README.md

```diff
@@ -1,34 +1,22 @@
-## Task 3
-
 ## Description
 
-Create separate github repo with the following pattern(cobol case): `golang-training-YOUR-DATABASE-CHEMA-NAME`, Example: golang-training-theater.
+Application for mantaining hospital with such operations as in table below:
+
 
-Database schema names:
-- Theater: `@Alexandr59`
-- Hospital: `@DaniilDomaskanov`
-- Shop: `@dimaxdqwerty`
-- Enterprise: `@jeka2708`
-- Library: `@PutskouDzmitry`
-- University: `@tonymontanapaffpaff`
-- Social network: `@VSiniak12`
-- Restaurant: `@miladouski`
+|             Path            | Method | Description                           | Body example                                                                                                                                                                                                                     |
+|:---------------------------:|--------|---------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| /doctors                   | GET    | get all doctors                      | {"DoctorId":3,"FirstName":"Victor","LastName":"Pashkevich","DateOfBirth":"2001-07-25","Salary":"570.14","CurrentBusyState":false,"GenderId":2,"SpecialityId":5},{"DoctorId":4,"FirstName":"Eva","LastName":"Dushkevich","DateOfBirth":"2002-10-22","Salary":"790.77","CurrentBusyState":true,"GenderId":1,"SpecialityId":1},{"DoctorId":5,"FirstName":"Nikita","LastName":"Miladovski","DateOfBirth":"1995-04-18","Salary":"680.56","CurrentBusyState":false,"GenderId":2,"SpecialityId":5},{"DoctorId":2,"FirstName":"Dmitry","LastName":"Putkov","DateOfBirth":"1999-02-13","Salary":"5000.54","CurrentBusyState":true,"GenderId":2,"SpecialityId":4},{"DoctorId":1,"FirstName":"Danik","LastName":"Voinov","DateOfBirth":"2000-09-12","Salary":"1250.12","CurrentBusyState":false,"GenderId":2,"SpecialityId":1} |
```

Comment:
could you show json body in the following way:
```json
Here goes your json object definition
```
I took this answer from here: https://stackoverflow.com/questions/14901245/how-to-style-a-json-block-in-github-wiki